htdocs/fourn/facture/card.php

```diff
@@ -885,7 +885,7 @@
 		}
 
 		// Standard invoice or Deposit invoice, created from a Predefined template invoice
-		if ((GETPOST('type') == FactureFournisseur::TYPE_STANDARD || GETPOST('type') == FactureFournisseur::TYPE_DEPOSIT) && GETPOST('fac_rec', 'int') > 0) {
```

Comment:
On this line it is simply a reordering of the conditions + using the variable instead of re-calling GETPOST